tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/ExtensionsMetadataGenerator.cs

```diff
@@ -90,6 +90,11 @@ public static bool IsWebJobsStartupAttributeType(TypeReference attributeType, Co
                     return true;
                 }
 
+                if (currentAttributeType.FullName.StartsWith("System", StringComparison.OrdinalIgnoreCase))
```

Comment:
Are we okay with only checking the name? Anyone can write a type in the `System` namespace. Will that cause any issues?

I am not that familiar with Mono, but I wonder if there is a stricter way to verify the assembly? But also, Is that effort even worth it when your current check will get the job done.

Reply:
Checking for System namespace is fine, because all we are looking for is webjobs attributes. We should be fine as long as we don't write to system or namespace for webjobs attributes.